code/modules/mob/living/simple_animal/bot/cleanbot.dm

```diff
@@ -33,23 +33,91 @@
 	var/next_dest
 	var/next_dest_loc
 
+	var/obj/item/weapon
+	var/weapon_orig_force = 0
+
+	var/list/stolen_valor
+
+
+	var/orig_name
+
+	var/list/officers = list("Admiral", "Spec Ops Officer", "Captain", "Head of Personnel", "Death Commando", "Head of Security")
+
+	var/list/command = list("Admiral" = "Admiral", "Spec Ops Officer" = "Col.", "Captain" = "Cpt.","Head of Personnel" = "Lt.")
+	var/list/security = list("Death Commando" = "Trooper", "Head of Security" = "Maj.", "Warden" = "Sgt.", "Detective" =  "Det.", "Security Officer" = "Officer")
+	var/list/engineering = list("Chief Engineer" = "Chief Engineer", "Station Engineer" = "Engineer", "Atmospherics Technician" = "Technician")
+	var/list/medical = list("Chief Medical Officer" = "C.M.O.", "Medical Doctor" = "M.D.", "Chemist" = "Pharm.D.")
+	var/list/research = list("Research Director" = "Ph.D.", "Roboticist" = "M.S.", "Scientist" = "B.S.")
```

Comment:
Excuse me, are you saying that greasy wrench monkeys are better educated than enlightened nanotechnology enthusiasts?

Reply:
Quiet before I bust that down to B.A.